plugins/ReactTestComponent.js

```diff
@@ -0,0 +1,69 @@
+var reactTestInstance = Symbol.for('react.test.json');
+
+function getIndentString(indent) {
```

Comment:
I wrote this for the previous version of pretty-format so there was no boundIndent passed, I just didn't change this part yet